composer.json

```diff
@@ -14,6 +14,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.0",
+        "ext-json": "*",
```

Comment:
Although this is an edge case because most PHP instances will have json enabled. 
The best practice for this would be to add this as a dev requirement as it is required for unit tests, and add a `suggest` for `ext-json` in composer.json
```json
 "suggest": {
        "ext-json": "Required for Json filters"
    }
```

Reply:
Alright, I'll make that change.